### PR TITLE
Fix cli flag orders (long, short)

### PR DESCRIPTION
**Pull request by rht**
cc: @chriscool 

- [x] There is `ipfs ls --headers / ipfs ls --` as well.

### CODE REVIEW

**Comment by whyrusleeping** [comment]
LGTM

**Comment by chriscool** [comment]
Yeah LGTM too. Thanks @rht

File: core/commands/id.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ ipfs id supports the format option for output with the following keys:
 		cmds.StringArg("peerid", false, false, "peer.ID of node to look up").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("f", "format", "optional output format"),
+		cmds.StringOption("format", "f", "optional output format"),
```
Comment (jbenet) [inline core/commands/id.go]: is this rendering correctly? should render as:

```
-f, --format
```

Comment (rht) [inline core/commands/id.go]: Yes it does, on this branch,
```
$ ipfs id --help

    ipfs id [<peerid>] - Show IPFS Node ID info

ARGUMENTS:

    [<peerid>] - peer.ID of node to look up

OPTIONS:

    -f, --format string - optional output format

DESCRIPTION:

    Prints out information about the specified peer,
    if no peer is specified, prints out local peers info.
    
    ipfs id supports the format option for output with the following keys:
    <id> : the peers id
    <aver>: agent version
    <pver>: protocol version
    <pubkey>: public key
    <addrs>: addresses (newline delimited)
```

**Pull request merged**